cis/Issues/complianceCheck/description.xml

```diff
@@ -80,6 +60,26 @@
 				<nl>Je geslacht</nl>
 			</Description>
 		</Attribute>
+		<Attribute id="nationality" optional="true" displayIndex="4" displayHint="yesno">
+			<Name>
+				<en>Dutch nationality</en>
+				<nl>Nederlandse nationaliteit</nl>
+			</Name>
+			<Description>
+				<en>Whether you have the dutch nationality</en>
+				<nl>Of je de Nederlandse nationaliteit bezit</nl>
+			</Description>
+		</Attribute>
+		<Attribute id="digidlevel" displayIndex="5">
+			<Name>
+				<en>Assurance level</en>
+				<nl>Betrouwbaarheidsniveau</nl>
+			</Name>
+			<Description>
+				<en>The assurance level with which your identity was verified</en>
+				<nl>Het betrouwbaarheidsniveau waarmee je identiteit is vastgesteld</nl>
+			</Description>
+		</Attribute>
```

Comment:
It's currently not supported to remove existing attributes. The only possible way now is to add `optional="true"` to the attributes you want to get rid of, and then you can skip them on issuance. New attributes should be added to the end of the XML list. The visible order can be changed using `displayIndex`.